day_6/exercises/burrito.rb

```diff
@@ -1,19 +1,48 @@
-# Add the following methods to this burrito class and 
+# Add the following methods to this burrito class and
 # call the methods below the class:
 # 1. add_topping
 # 2. remove_topping
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
+  #attr_accessor :protein, :base, :toppings
+  attr_accessor :protein, :base, :toppings
+
   def initialize(protein, base, toppings)
-    @protein  = protein
-    @base     = base
+    @protein = protein
+    @base = base
+    @toppings = toppings
+  end
+
+  def info # kept trying to define this method with the variables after the method name, figured out that doesn't work.
+    p "My burrito has #{self.protein}, #{self.base}, and #{self.toppings}."
+  end
+
+  def add_topping(toppings)
```

Comment:
I would only expect this to add a single topping. How would you change this and calling the method below?